lib/composer.js

```diff
@@ -31,13 +31,15 @@ class Composer {
       mutationTypeName = 'Mutation',
       subscriptionTypeName = 'Subscription',
       subgraphs = [],
+      onSubgraphError = ({ error }) => { throw error },
```

Comment:
I liked how you previously had `noop` as a top level reusable function. Can we do the same thing here?